templates/rhpam73-kieserver-externaldb.yaml

```diff
@@ -301,7 +301,7 @@ parameters:
   value: mykeystorepass
   required: false
 - displayName: KIE Server Bypass Auth User
-  description: KIE server bypass auth user. (Sets the org.kie.server.bypass.auth.user system property)
+  description: Allow API requests to KIE server without user authentication. (Sets the org.kie.server.bypass.auth.user system property)
```

Comment:
I think it should be changed to:

"Allows the KIE server to bypass the authenticated user for task related operations e.g. queries. (Sets the org.kie.server.bypass.auth.user system property)"

Gotten from https://docs.jboss.org/drools/release/7.18.0.Final/drools-docs/html_single/ .